### PR TITLE
Functional assay update

### PR DESCRIPTION
**Pull request by zfisch**
Introduces new functional assay data selected by the ENIGMA Consortium and major changes to the functional assay tile.

### CODE REVIEW

File: pipeline/field_metadata.tsv [hunk 1]
```diff
@@ -105,206 +105,236 @@ BX_ID_BIC	Internal	#SKIP	"Arbitrary, unique identifiers for BIC submissions used
...
-Allele_frequency_genome_GnomADv3	Allele Frequency	#COPY	"Overall allele frequency (for genomes) according to all studied populations, per gnomAD v3"
-BX_ID_GnomADv3	Internal	#SKIP	"Arbitrary, unique identifier used by BRCA Exchange for the purpose of variant aggregation"
-HGVS_Nucleotide_Findlay_BRCA1_Ring_Function_Scores	Nomenclature	#SKIP	HGVS nucleotide nomenclature for the variant in the Findlay functional assay	c.-10A>T
-Log_RNA_Depletion_Findlay_BRCA1_Ring_Function_Scores	Evidence	#COPY	Log ratio comparing RNA expression with and without the variant	-0.006252529515
-Functional_Enrichment_Score_Findlay_BRCA1_Ring_Function_Scores	Evidence	#COPY	"Functional assay results for variants in critical domains of BRCA1, obtained using saturation genome editing; provided by Starita lab (see Findlay et al. 2018)"	-0.107071247
-BX_ID_Findlay_BRCA1_Ring_Function_Scores	Internal	#SKIP	"Arbitrary, unique identifier used by BRCA Exchange for the purpose of variant aggregation"	152
+faf95_popmax_genome_GnomAD	Allele Frequency	faf95_popmax_genome_GnomADv2	The gnomAD genome population frequency that is most significant with a 95% confidence threshold (gnomAD 2.1)
```
Comment (zfisch) [inline pipeline/field_metadata.tsv:108]: double check that these changes are safe

File: pipeline/functional_assays/convert_functional_assay_tsv_to_vcf.py [hunk 2]
```diff
@@ -81,15 +82,26 @@ def main():
...
-        cdna_hgvs_str = ref_seq + ":" + nuleotide
-
-        print('{0}\t{1}\t{2}\t{3}\t{4}\t.\t.\t{5}'.format(parsedLine[fieldIdxDict['chr']],
-                                                          parsedLine[fieldIdxDict['pos_hg19']],
+        try:
+            var_hgvs = hgvs_wrapper.hgvs_parser.parse(parsedLine[3])
+            var_hgvs_norm = hgvs.normalizer.Normalizer(hgvs_wrapper.hgvs_dp, shuffle_direction=5).normalize(var_hgvs)
+            v = variant_utils.VCFVariant.from_hgvs_obj(var_hgvs_norm, seq_fetcher37)
+        except hgvs.exceptions.HGVSParseError as e:
+            print(e)
```
Comment (zfisch) [inline pipeline/functional_assays/convert_functional_assay_tsv_to_vcf.py:90]: log instead of print

File: pipeline/utilities/variantsBySource.py [hunk 3]
```diff
@@ -24,7 +24,7 @@ def main():
                               "ExAC": 0,
                               "BIC": 0,
                               "ESP": 0,
-                              "Findlay_BRCA1_Ring_Function_Scores": 0,
```
Comment (zfisch) [inline pipeline/utilities/variantsBySource.py:27]: maybe include both for historical reasons?

File: pipeline/workflow/tarball_files_discard_list.txt [hunk 4]
```diff
@@ -36,7 +36,8 @@
 ./release/artifacts/rightESP
 ./release/artifacts/rightExAC
 ./release/artifacts/rightexLOVD
-./release/artifacts/rightFindlay_BRCA1_Ring_Function_Scores
+./release/artifacts/rightENIGMA_BRCA12_Functional_Assays
+./release/artifacts/
```
Comment (zfisch) [inline pipeline/workflow/tarball_files_discard_list.txt:40]: mistake?

File: pipeline/utilities/variantsBySource.py (hunk 3 above)
Comment (melissacline) [reply pipeline/utilities/variantsBySource.py:27]: Good point.  Something to consider is the impact of these changes on variant history.  For the variants that are covered by the Findlay assay, the same assay results should be in the variant record from the ENIGMA assays moving forward, but looking back at legacy data is another matter.  Maybe including both is cleaner for managing the legacy data.

Comment (zfisch) [reply pipeline/utilities/variantsBySource.py:27]: yea, we'll want to review this in beta. i anticipate issues viewing legacy variants and we'll need to decide what's worth supporting

**Pull request merged**